codespell_lib/tests/test_basic.py

```diff
@@ -100,15 +100,15 @@ def test_basic(tmpdir, capsys):
     os.remove(fname)
 
     with open(op.join(d, 'bad.txt'), 'w') as f:
-        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd')
-    assert cs.main(d) == 4
+        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd\nabandonned\rAbandonned\r\nABANDONNED \nAbAnDoNnEd')  # noqa: E501
```

Comment:
```suggestion
        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd\nabandonned\rAbandonned\r\n ABANDONNED \nAbAnDoNnEd')  # noqa: E501
```

Reply:
Good call, but I've done it on a separate, currently unused test, rather than on the trailing whitespace one.
```suggestion
        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd\nabandonned\rAbandonned\r\nABANDONNED \n AbAnDoNnEd')  # noqa: E501
```